crml/staking/reward-curve/src/log.rs

```diff
@@ -1,3 +1,20 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
```

Comment:
we might not be able to claim this one until we actually modify it

Reply:
yep changed the text into Parity's copyright text